structure_tests/structure_test.go

```diff
@@ -88,39 +87,48 @@ func Parse(t *testing.T, fp string) (StructureTest, error) {
 	if !ok {
 		return nil, errors.New("Error encountered when type casting Structure Test interface")
 	}
-	tests.SetDriverImpl(driverImpl, imageName)
+	tests.SetDriverImpl(driverImpl, imagePath)
 	return tests, nil
 }
 
 var configFiles arrayFlags
-var imageName, driver string
-var driverImpl func(string) drivers.Driver
+
+var imagePath, driver string
+var driverImpl func(string) (drivers.Driver, error)
 
 func TestMain(m *testing.M) {
-	flag.StringVar(&imageName, "image", "", "path to test image")
+	flag.StringVar(&imagePath, "image", "", "path to test image")
 	flag.StringVar(&driver, "driver", "docker", "driver to use when running tests")
 
+	var logLevel string
+	flag.StringVar(&logLevel, "logLevel", "0", "test")
```

Comment:
What's this for?

Reply:
this is for logging in vendored code that uses the standard `-v` flag, i.e. container-diff.

Reply:
Not sure I understand. container-diff also uses glog, so it should be registering it's own flag automatically. Are you just trying to make it so one flag (logLevel) sets the -v parameter as well?

Reply:
well container-diff uses the `-v` flag for log level, but the structure tests use their own `-test.v` flag since they're run with `go test`. so I originally had to set the other flag to get logging to show up from container-diff code....but it looks like that's not the case anymore. I'll remove this